database_schema.md

```diff
@@ -136,6 +136,87 @@ seat_id: {
 }
 ```
 
+### Minutes Item
+```
+minutes_item: {
+    minutes_item_id: str
+    name: str
+    title: str
+    description: str
+    matter: {
+      matter_id: str
+      matter_name: str
+      matter_title: str
+    }
+    external_source_id: str
+    created: datetime
+}
+```
+
+### Event Minutes Item
+```
+event_minutes_item: {
+    event_minutes_item_id: str
+    event_id: str
+    minutes_item: {
+      minutes_item_id: str
+      minutes_item_name: str
+    }
+    index: int
+    decision: str
+    matter: {
+      matter_id: str
+      matter_name: str
+      matter_title: str
+    }
+    votes: [
+        {
+            vote_id: str
+            person_id: str
+            person_name: str
+            vote_decision: str
+        }
+    ]
+    files: [
+        {
+           event_minutes_item_file_id: str //can probably just shortened the field name to just id
+           name: str
+           uri: str
+        }
+    ]
+}
+```
+
+### Vote
+```
+vote: {
+    matter: {
+        matter_id: str
+        matter_title: str
+        matter_name: str
+        matter_type_name: str
+    }
+    event: {
+        event_id: str
+        event_body_name: str
+        event_datetime: datetime
+        event_minutes_item_id: str
+    }
+    minutes_item: {
+        minutes_item_id: str
+        minutes_item_decision: str
```

Comment:
I understand what you are trying to do here, but `minutes_item` doesn't have a `decision` field. We should use `event_minutes_item_decision`?
```
    event_minutes_item: {
        event_minutes_item_id: str
        event_minutes_item_decision: str
    }
```
and remove `event_minutes_item_id: str` in line 203?

Reply:
Agree. Remove `event_minutes_item_id: str` on 203.

I think we could realistically have both `minutes_item` and `event_minutes_item` blocks. So instead of `decision` here you could just have `name`? Then the `event_minutes_item` can have `id`, `name`, and `decision`?

Reply:
I don't think we need `name` for  `minutes_item` block or `event_minutes_item` block because since we already have `matter_name`?

Reply:
Ehhhh I see your point. I feel like there is value in at least having `event_minutes_item`? Like, that is what a vote is for; a vote occurs on an event minutes item. Just seems odd to leave it off? Haha idk maybe it's too much info to store a single vote but I feel like it would be more natural.

Reply:
I agree we definitely need at least an `event_minutes_item` block. I just think that we don't need  `event_minutes_item.name` since we already have `matter.matter_name`.

Reply:
Ahhhhh. Yes agree.